rotkehlchen/chain/optimism/modules/hop/constants.py

```diff
@@ -0,0 +1,81 @@
+from typing import Final
+
+from rotkehlchen.chain.evm.constants import ZERO_ADDRESS
+from rotkehlchen.chain.evm.decoding.hop.structures import HopBridgeEventData
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.constants.assets import A_ETH
+
+ETH_BRIDGE: Final = string_to_evm_address('0x83f6244Bd87662118d96D9a6D44f09dffF14b30E')
+ETH_AMM_WRAPPER: Final = string_to_evm_address('0x86cA30bEF97fB651b8d866D45503684b90cb3312')
+
+USDC_BRIDGE: Final = string_to_evm_address('0xa81D244A1814468C734E5b4101F7b9c0c577a8fC')
+USDC_AMM_WRAPPER: Final = string_to_evm_address('0x2ad09850b0CA4c7c1B33f5AcD6cBAbCaB5d6e796')
+
+USDT_BRIDGE: Final = string_to_evm_address('0x46ae9BaB8CEA96610807a275EBD36f8e916b5C61')
+USDT_AMM_WRAPPER: Final = string_to_evm_address('0x7D269D3E0d61A05a0bA976b7DBF8805bF844AF3F')
+
+DAI_BRIDGE: Final = string_to_evm_address('0x7191061D5d4C60f598214cC6913502184BAddf18')
+DAI_AMM_WRAPPER: Final = string_to_evm_address('0xb3C68a491608952Cb1257FC9909a537a0173b63B')
+
+HOP_BRIDGE: Final = string_to_evm_address('0x03D7f750777eC48d39D080b020D83Eb2CB4e3547')
+HOP_AMM_WRAPPER: Final = ZERO_ADDRESS
+
+SNX_BRIDGE: Final = string_to_evm_address('0x16284c7323c35F4960540583998C98B1CfC581a7')
+SNX_AMM_WRAPPER: Final = string_to_evm_address('0xf11EBB94EC986EA891Aec29cfF151345C83b33Ec')
+
+SUSD_BRIDGE: Final = string_to_evm_address('0x33Fe5bB8DA466dA55a8A32D6ADE2BB104E2C5201')
+SUSD_AMM_WRAPPER: Final = string_to_evm_address('0x29Fba7d2A6C95DB162ee09C6250e912D6893DCa6')
+
+RETH_BRIDGE: Final = string_to_evm_address('0xA0075E8cE43dcB9970cB7709b9526c1232cc39c2')
+RETH_AMM_WRAPPER: Final = string_to_evm_address('0x19B2162CA4C2C6F08C6942bFB846ce5C396aCB75')
+
+BRIDGES: Final = {
```

Comment:
as I mentioned in discord move all constants into the bridges and if something is not used twice remove as a constant. in all files.